tree/ntuple/v7/inc/ROOT/RNTupleModel.hxx

```diff
@@ -172,6 +173,23 @@ public:
                                       std::function<std::string(const std::string &)> mapping);
    };
 
+   /// Provides mutable access to the field tree of the model. This is necessary to commit clusters during writing
+   /// and to set the on-disk field IDs when connecting a model to a page source or sink.
+   /// Attorney-Client limited friendship pattern.
```

Comment:
Hm, TIL about this pattern... Would it instead be feasible to have both
```c++
const RFieldZero &GetFieldZero() const { return *fFieldZero; }
RFieldZero &GetFieldZero() { return *fFieldZero; }
```
and ensure that all places that need have a non-`const` version of the the model?